### PR TITLE
stock_purchase_analytic_plans converted to v8

### PR DESCRIPTION
**Pull request by levkar**
This addon haven't been ported to v8 so I get it working.

### CODE REVIEW

**Comment by nhomar** [comment]
@hbto can you review if this is valid in order to merge it please.

@levkar thanks a lot for contribute.

**Comment by levkar** [comment]
@nhomar you're welcome. thanks for your contributions.

i made a change to insert it into one of the lines (debit on outgoing and credit in incoming). if we have both, they cancel each other so we want only one of them.

File: stock_purchase_analytic_plans/model/stock.py [hunk 1]
```diff
@@ -22,20 +22,27 @@
...
-            debit_line_vals[
-                'analytics_id'] = move.purchase_line_id.analytics_id.id
-            credit_line_vals[
-                'analytics_id'] = move.purchase_line_id.analytics_id.id
+            if move.location_dest_id.usage == 'internal':
+                debit_line_vals[
+                    'analytics_id'] = purchase_line_id.analytics_id.id
+            else:
+                credit_line_vals[
+                    'analytics_id'] = purchase_line_id.analytics_id.id
```
Comment (hbto) [inline stock_purchase_analytic_plans/model/stock.py:46]: @nhomar I am ok with @levkar changes

You can merge this

Regards

**Comment by hbto** [comment]
@levkar could you fix the issue in travis in order to merge it to main-branch?

Regards

**Comment by levkar** [comment]
taken care of

**Pull request merged**